iothub/device/src/Exceptions/IotHubClientErrorCode.cs

```diff
@@ -9,9 +9,9 @@ namespace Microsoft.Azure.Devices.Client
     public enum IotHubClientErrorCode
     {
         /// <summary>
-        /// The request completed without exception.
+        /// Used when the error code returned by the hub is unrecognized. If encountered, please report the issue so it can be added here.
         /// </summary>
-        Ok,
+        Unknown,
```

Comment:
Changing the default case from "Ok" to "Unknown". 
IotHubClientErrorCode being used only via IotHubClientException, a value of "Ok" didn't seem right.

Reply:
Unless the Hub actually returns the value 'Ok', but I don't know why it would for an error.